metricbeat/mb/module/publish.go

```diff
@@ -21,8 +21,11 @@ import (
 	"sync"
 
 	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/libbeat/logp"
 )
 
+const staticModuleSink = "@static"
```

Comment:
Here the module would be "registered" or "native", we call them "static" when they come from the configuration in files, vs. dynamic that could come from dynamic configurations as autodiscover.
```suggestion
const registeredModuleSink = "@registered"
```

We are also starting to refer to registered modules as "inputs", and this will likely be the terminology in the future (the `inputs` in the manifest comes from that).

We may need to review the use of this terminology around these changes.

Reply:
Refactored. Not relevant anymore.